commit 0386af9af8ff3f2708263fe7d7e47ae63b5cba41
Author: lexicalmathical <lexicalmathical@gmail.com>
Date:   Thu Dec 18 16:00:29 2025 +0800

    use gemini flash

diff --git a/backend/config.py b/backend/config.py
index 68dfa73..0e0dc29 100644
--- a/backend/config.py
+++ b/backend/config.py
@@ -9,6 +9,7 @@ MODEL = "deepseek-v3.2"
 MODEL = "gemini-3-pro-preview"
 MODEL = "mistral-small-creative"
 MODEL = "claude-sonnet-4.5"
+MODEL = "gemini-3-flash-preview"
 
 # @@@ Sparsity control
 MAX_VOICES = 5
