commit 6a134c937fb4c3152fdc4e3d66d648855142b72d
Author: lexicalmathical <lexicalmathical@gmail.com>
Date:   Tue Dec 16 16:17:44 2025 +0800

    use a smaller image

diff --git a/frontend/public/login-banner.jpg b/frontend/public/login-banner.jpg
new file mode 100644
index 0000000..472f9d0
Binary files /dev/null and b/frontend/public/login-banner.jpg differ
diff --git a/frontend/public/login-banner.png b/frontend/public/login-banner.png
deleted file mode 100644
index 23c7d41..0000000
Binary files a/frontend/public/login-banner.png and /dev/null differ
diff --git a/frontend/src/App.tsx b/frontend/src/App.tsx
index 52d49a4..9544852 100644
--- a/frontend/src/App.tsx
+++ b/frontend/src/App.tsx
@@ -885,7 +885,7 @@ export default function App() {
   }
 
   // @@@ Show auth screen if not authenticated
-  const loginBannerUrl = `${import.meta.env.BASE_URL}login-banner.png`;
+  const loginBannerUrl = `${import.meta.env.BASE_URL}login-banner.jpg`;
 
   if (!isAuthenticated) {
     return (
