commit 587a5eaaa09e873cc1a4f2a9577f3c17b6ff4763
Author: lexicalmathical <lexicalmathical@gmail.com>
Date:   Wed Dec 3 18:49:09 2025 +0800

    edit author name

diff --git a/frontend/src/components/AboutView.tsx b/frontend/src/components/AboutView.tsx
index 102bf06..525730e 100644
--- a/frontend/src/components/AboutView.tsx
+++ b/frontend/src/components/AboutView.tsx
@@ -95,7 +95,7 @@ export default function AboutView() {
                 marginBottom: 4,
                 fontFamily: '-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, sans-serif'
               }}>
-                Lexical Mathical
+                F2J
               </div>
               <div style={{
                 fontSize: 14,
